### PR TITLE
feat: add envs to docker-compose

### PR DESCRIPTION
**Pull request by ratik**
Changes:
- added missing env variables to relayer config, making it not crash on `make start-cosmopark`;
- configured relayer to use `demowallet3` instead of `demowallet1` as its signing key, this will allow us to freely use demowallet1 in tests, without it being occupied by relayer, which prevented us to issue transactions.

### CODE REVIEW

**Review by foxpy — DISMISSED** [review]

**Review by foxpy — DISMISSED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -53,6 +53,8 @@ services:
       - RELAYER_TARGET_CHAIN_OUTPUT_FORMAT=json
       - RELAYER_TARGET_CHAIN_SIGN_MODE_STR=direct
       - RELAYER_REGISTRY_ADDRESSES=
+      - RELAYER_NEUTRON_CHAIN_ALLOW_KV_CALLBACKS=1
+      - RELAYER_ALLOW_TX_QUERIES=1
```
Comment (quasisamurai) [inline docker-compose.yml]: why `1`, not `true`?

Comment (ratik) [reply docker-compose.yml]: dunno, bc it works 😅
shall I change it to `true`?

Comment (foxpy) [reply docker-compose.yml]: Let's hope next reviewer wouldn't ask us to rename it back to 1 :v

**Review by foxpy — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (quasisamurai) [reply docker-compose.yml]: I believe `true` is kinda more explicit, due the type is bool

**Review by quasisamurai — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (sotnikov-s) [reply docker-compose.yml]: hey guys we can save a couple of bytes by dropping the superfluous `true` and using simple `1`

File: docker-compose.yml [hunk 2]
```diff
@@ -53,6 +53,8 @@ services:
       - RELAYER_TARGET_CHAIN_OUTPUT_FORMAT=json
       - RELAYER_TARGET_CHAIN_SIGN_MODE_STR=direct
       - RELAYER_REGISTRY_ADDRESSES=
+      - RELAYER_NEUTRON_CHAIN_ALLOW_KV_CALLBACKS=true
+      - RELAYER_ALLOW_TX_QUERIES=true
```
Comment (sotnikov-s) [inline docker-compose.yml:57]: out of scope, but I don't like the way we've designed these variables. why one of them is inside the neutron chain's config, and the other one is inside the relayer's config? I think they both should be on the relayer's level bc they're not related to the neutron chain, they're about the relayer configuration. do you agree? if yes, would it hurt if we fix it here?

File: docker-compose.yml (hunk 1 above)
Comment (ratik) [reply docker-compose.yml]: nope

File: docker-compose.yml (hunk 2 above)
Comment (ratik) [reply docker-compose.yml:57]: I think it's really out of scope of this PR. So let's have separate task. Would you mind creating this one

**Review by pr0n00gler — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (sotnikov-s) [reply docker-compose.yml]: just a joke

File: docker-compose.yml (hunk 2 above)
Comment (sotnikov-s) [reply docker-compose.yml:57]: sure

**Review by sotnikov-s — APPROVED** [review]

**Pull request merged**